src/cider/nrepl/middleware/slurp.clj

```diff
@@ -43,7 +43,8 @@
   `[type {:as attrs}]`. This method normalizes RFC
   compliant content-types to this form."
   [^String content-type]
-  (if-let [match (re-find content-type-pattern content-type)]
+  (if-let [match (and content-type
```

Comment:
Shouldn't this `nil` check be handled in the caller of `normalize-content-type`? I think it's weird to expect to receive a `nil` argument.

Reply:
Yeah. Makes sense.